functions/snippets/src/main/java/com/example/functions/CorsEnabled.java

```diff
@@ -33,7 +33,7 @@
   public void service(HttpRequest request, HttpResponse response)
       throws IOException {
     // Set CORS headers for preflight requests
-    if (request.getMethod().equals("OPTIONS")) {
+    if ("OPTIONS".equals(request.getMethod())) {
```

Comment:
I'd be interested in hearing a bit more about this, it's a tiny bit surprising, but probably correct.